dali/aux/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_subdirectory(optical_flow)
+
```

Comment:
I think you need to put
```
set(DALI_SRCS ${DALI_SRCS} PARENT_SCOPE)
```
as well to propagate DALI_SRCS from the `optical_flow` dir.

Reply:
Done